src/test/resources/com/puppycrawl/tools/checkstyle/astprinter/InputFullOfBlockCommentsCR.java

```diff
@@ -0,0 +1 @@
+/*0*//*1*/package/*2*/ com.puppycrawl.tools.checkstyle.astprinter/*11*/;/*12*/
/*13*/
/*14*/public/*15*/ class /*16*/InputFullOfBlockCommentsCR /*49*/{/*17*/
	/*18*/
	/*19*/public/*20*/ static/*
21
*/ String/*22*/ main/*23*/(/*24*/String/*25*/[/*26*/]/*27*/ args/*28*/)/*29*/ {/*30*/
		/*31*/String /*32*/line /*33*/= /*34*/"/*I'm NOT comment*/blabla"/*35*/;/*36*/
		/*37*/String/*38*/./*  39  */CASE_INSENSITIVE_ORDER/*40*/./*41*/equals/*42*/(/*43*/line/*44*/)/*45*/;/*46*/
		Integer[] array = null;
		/*50*/for/*51*/ (/*52*/Integer/*53*/ i/*54*/:/*55*/ array/*56*/)/*57*/ {/*58*/
		    /*59*/
		}/*60*/
		return line;
	}/*47*/
}/*48*/
/*61*/

```

Comment:
do we really need this file be that big ?
if you focus on some CR  catch, file could be probably minimized.
You can add new line symbol at the end of file, to avoid suppression - it is not a point of testing here.